src/RL/nn_structures/cpnn.jl

```diff
@@ -53,3 +53,44 @@ function (nn::CPNN)(states::BatchedDefaultTrajectoryState)
         return output
     end
 end
+
+# Overloads the Base.string() function for storing parameters of the neural networks associated to experiments.
+function Base.string(nn::CPNN)
```

Comment:
Do we really need this as long as CPNN will be partially refacto ?

Reply:
It has nothing to do with the reward but it is really useful to keep track of the neural networks we use when we do experiments (the returned string is saved in `params.json` files). For performing experiments on the new heterogeneous pipeline we have duplicated CPNN so I think we should keep it like this for the moment.